sdk/js/src/Request.ts

```diff
@@ -66,12 +70,18 @@ export const getConfigJson = async (
         const res = await getWithTimeout(url, requestConfig, 5000)
         return await res.json()
     } catch (e) {
+        const error = e as ResponseError
```

Comment:
this isnt a valid type assertion, error could be anything (like what about when json fails to parse?)

Reply:
will refactor 👍